openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/storages/WheelchairAttributesGraphStorage.java

```diff
@@ -69,20 +70,29 @@ public WheelchairAttributesGraphStorage()  {
 		trackTypeEncoder = new EncodedValueOld("tracktype", shift, 3, 1, 0, TRACK_TYPE_MAX_VALUE);
 		shift += trackTypeEncoder.getBits();
 
-		hasInclineEncoder = new EncodedValueOld("hasIncline", shift, 1, 1, 0, 1);
-		shift += 1;
 		inclineEncoder = new EncodedValueOld("incline", shift, 5, 1, 0, INCLINE_MAX_VALUE);
 		shift += inclineEncoder.getBits();
 
-		hasKerbHeightEncoder = new EncodedValueOld("hasKerbHeight", shift, 1, 1, 0, 1);
-		shift += 1;
 		kerbHeightEncoder = new EncodedValueOld("kerbHeight", shift, 4, 1, 0, KERB_MAX_VALUE);
 		shift += kerbHeightEncoder.getBits();
 
 		widthEncoder = new EncodedValueOld("width", shift, 5, 10, 0, WIDTH_MAX_VALUE);
 		shift += widthEncoder.getBits();
 
 		sideFlagEncoder = new EncodedValueOld("side", shift, 2, 1,0,2);
+		shift += sideFlagEncoder.getBits();
+
+		hasKerbHeightEncoder = new EncodedValueOld("hasKerbHeight", shift, 1, 1, 0, 1);
+		shift += 1;
+
+		hasInclineEncoder = new EncodedValueOld("hasIncline", shift, 1, 1, 0, 1);
```

Comment:
EncodedValueOld should not be used in new code

Reply:
see above